src/components/dataset/overview/DatasetOverviewPanel.jsx

```diff
@@ -178,6 +211,78 @@ function DatasetOverviewPanel(props) {
             <Typography variant="h6">Cloud Platforms:</Typography>
             {renderCloudPlatforms(dataset)}
           </Grid>
+          <Grid item xs={4}>
+            <Typography variant="h6">Dataset Id:</Typography>
+            <Typography>{dataset.id}</Typography>
```

Comment:
This is totally OOS, but it would be nice if we gave the option to copy UUIDs on click to minimize chances of transcription errors.